### PR TITLE
WIP: attempt to debug failing Windows build on AppVeyor

### PR DESCRIPTION
**Pull request by gillins**
xref: https://github.com/conda-forge/qt-feedstock/issues/26

Weird because it is working for me locally and previous AppVeyor builds are ok...

### CODE REVIEW

**Comment by ocefpaf** [comment]
Thanks for looking into this @gillins!

**Comment by gillins** [comment]
it seems that `configure.exe` has been "removed" at some stage during extraction of the tarball - see https://ci.appveyor.com/project/conda-forge/qt-feedstock/build/1.0.49/job/dbqo9xir2u9kaj2g#L364. 

This doesn't happen for me locally (yes I did clear out the source cache) - `configure.exe` is definitely there. Here is a previous AppVeyor build that succeeded: https://ci.appveyor.com/project/conda-forge/qt-feedstock/build/1.0.31/job/01g6s3if2pxkkrcc

Does anyone have any ideas what is going on? are *.exe files now being removed when they are untarred by AppVeyor or `conda-build`? Has something changed recently?

**Comment by ocefpaf** [comment]
> Has something changed recently

I will raise an issue with AppVeyor to ask. Could be a new safety measure or something...

**Comment by gillins** [comment]
@mingwandroid @ccordoba12 Any ideas on how to rebuild `configure.exe` on `qt4`? It seems to be removed by AppVeyor or conda (maybe something thinks it is a virus?). I had a play and rebuilding `configure.exe` seems to require an already `configure`d `qt4`. Have either of you tried this before?

**Comment by mingwandroid** [comment]
I haven't built Qt4 for about 3 years now, but as I remember, you just ran configure.bat and it would detect there not being a configure.exe and so built it?

**Comment by gillins** [comment]
There doesn't seem to be a configure.bat anymore... :cry:

**Comment by ccordoba12** [comment]
Great work @gillins! Things are passing now :-)

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,7 +1,11 @@
 set INCLUDE=%LIBRARY_INC%;%INCLUDE%
 set LIB=%LIBRARY_LIB%;%LIB%
 
-cd qt-everywhere-opensource-src-%PKG_VERSION%
+:: For some reason configure.exe is deleted on AppVeyor. Virus check?
+:: Anyway, it has been added to the recipe so copy it if it doesn't exist
+if not exist configure.exe (
+    copy %RECIPE_DIR%\configure.exe .
```
Comment (ocefpaf) [inline recipe/bld.bat]: Works for me :-)

**Pull request merged**